services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -43,11 +43,23 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar w
 
         when(projectRepository.deactivateProjectById(projectId)).thenReturn(Future(0))
         when(projectRepository.getProjectById(projectId)).thenReturn(Future(Some(project)))
-
         projectService.deactivateProjectById(projectId, userId).map { _ shouldBe project }
       }
     }
 
+    "updateProjectVersion" should {
+      "return correct future" taggedAs Service in {
```

Comment:
Let's make this message meaningful. At least we can change it to
```suggestion
      "succeed if repository returned successful result" taggedAs Service in {
```